includes/definitions/discovery/edgecos.yaml

```diff
@@ -5,3 +5,10 @@ modules:
               -
                   oid: cpuCurrentUti
                   num_oid: '.1.3.6.1.4.1.259.10.1.24.1.39.2.1.{{ $index }}'
+mib: ECS4610-50T-MIB
```

Comment:
You don't need the PHP version with this although it might be better to use the PHP version so we can shrink it all into one line so remove this.

Reply:
Specifying a the mib key twice will overwrite the first.

You can do this:
```yaml
oid: ECS4610-50T-MIB::cpuCurrentUti
value: cpuCurrentUti
```